grammar.js

```diff
@@ -690,20 +686,20 @@ module.exports = grammar({
       $.interpreted_string_literal
     ),
 
-    raw_string_literal: $ => token(seq(
+    raw_string_literal: $ => seq(
       '`',
       repeat(/[^`]/),
       '`'
-    )),
+    ),
```

Comment:
For some cases, it can be helpful to have the anonymous nodes here included in the tree.

Reply:
I still think we should treat strings and rune literals as single tokens, as opposed to having separate tokens for the quotes and the content.

Reply:
@maxbrunsfeld you're absolutely right, that's a good catch. I've reverted this change.